conneg-by-ap/index.html

```diff
@@ -841,6 +841,43 @@ <h3>Alternate Representations Data Model</h3>
       </p>
     </section>
   </section>
+  <section id="link-attributes">
+    <h2>Link Attributes</h2>
+      <section id="link-attributes-token">
+	<h3>Token</h3>
+	<p>
+	  When used in a <code>Link</code> header field,
+	  the attribute <code>token</code> is used to specify
+	  the mapping between the profile URI
+	  (specified using the <code>anchor</code> attribute)
+	  and a <a>token</a> used as an alternative to the full profile URI.
+	</p>
+	<p>
+	  It should be noted that the scope of the mapping between the token and the profile URI
+	  is per default limited to the current resource
+	  (i. e. the resource identified by the current request URI).
+	  Servers MAY use a larger scope for their mappings
+	  but clients should not depend on that
+	  unless the server documentation explicitly gives other instructions.
+	</p>
+	<p>
+	  The ABNF for the profile attribute's value is <code>token</code>,
```

Comment:
For the `token` attribute value?
Is `quoted-string`?

Is the Web Linking RFC not more appropriate here?